homework.md

```diff
@@ -44,70 +44,71 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
-
+A programming language used primarily by Web browsers to create a dynamic and interactive experience for the user
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+When people want to show theirs code to others
+People can see that what is the older and new versions of that programme
+Programmers can store their project confidently.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You can join and prepare to download any programme without any changes in main project
 
 ## Question 10 
 
-What happens when you clone a repostory?
+What happens when you clone a repository?
 
-It send it to a friend
+You can download any programme to your local computer without any changes in main project
 
```

Comment:
I'm not sure about this answer, I think it should be that the repository is copied to your local drive.